css/custom.scss

```diff
@@ -221,6 +225,45 @@ $link-active: $link-hover;
   }
 }
 
+.section {
+  margin: 2rem auto;
+  height: 70%;
+}
+.post {
```

Comment:
It would be good to merge the changes here with `.post` below

Reply:
How do you mean?

If I move the changes to here...

```
.post {
  @include mobile() {
```

...I think the styling will only be applied to the mobile view...no?

Reply:
If you move outside the @include it should be good